dts/bindings/mtd/adesto,at45d.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Adesto AT45D family DataFlash SPI Memory
```

Comment:
The compatible for this driver should be `atmel,at45` as it is in Linux.  Adesto is just a vendor conforming to that API.  The D part of the model identifier does not affect the definition.